test/deploy/kubernetes/destroy.sh

```diff
@@ -11,11 +11,20 @@ function print_header {
     echo "--------------------------------------------------------------------------"
 }
 
-# Delete Storage Service (Headless)
+# Delete Storage-Node Service (Headless)
```

Comment:
Here we should destroy storage or server/data nodes based on infra used...
Here you are only cleaning up data nodes

Reply:
Cleaning up all pods as of now, Currently, this script is working for both old as well as redefined pods flow.